code/modules/mob/dead/observer/observer.dm

```diff
@@ -648,7 +648,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	return 0
 
 /mob/dead/observer/can_admin_interact()
-	return check_rights(R_ADMIN, 0, src)
+	return client && client.observer_interact && check_rights(R_ADMIN, 0, src)

```

Comment:
How so? This allows admins to still turn of observer interacting if they do not need it enabled in a certain round (such as people that are prone to misclicking or use clicking to travel around the map as an observer instead of WASD).